docs/content/docs/connectors/datastream/pulsar.md

```diff
@@ -417,4 +417,16 @@ If you have a problem with Pulsar when using Flink, keep in mind that Flink only
 and your problem might be independent of Flink and sometimes can be solved by upgrading Pulsar brokers,
 reconfiguring Pulsar brokers or reconfiguring Pulsar connector in Flink.
 
+### Source stop reading records for about 10s when data volume is small
```

Comment:
Will you also add this section to the Chinese documentation?

Reply:
Sure.